automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/workspace/WorkspaceOwnerSpec.scala

```diff
@@ -30,53 +26,12 @@ class WorkspaceOwnerSpec extends FreeSpec with Matchers
 
 
   "A user" - {
-    "with a billing project" - {
-      "should be able to create a workspace" taggedAs Tags.SmokeTest in {
-        val user = UserPool.chooseStudent
-        implicit val authToken: AuthToken = user.makeAuthToken()
-        withCleanBillingProject(user) { billingProject =>
-          withWebDriver { implicit driver =>
-            withSignIn(user) { listPage =>
-              val workspaceName = "WorkspaceSpec_create_" + randomUuid
-              register cleanUp api.workspaces.delete(billingProject, workspaceName)
-              val detailPage = listPage.createWorkspace(billingProject, workspaceName)
-
-              detailPage.validateWorkspace shouldEqual true
-
-              listPage.open
-              listPage.hasWorkspace(billingProject, workspaceName) shouldBe true
-            }
-          }
-        }
-      }
-
-      "should be able to clone a workspace" in {
```

Comment:
redundant test. workspace clone feature covered by other tests, such as `should be cloned without copying the published status` in `PublishSpec`

Reply:
Alternative suggestion... keep this test and move the `PublishSpec` test to orchestration. I haven't read the code for `"should be cloned without copying the published status"` but the name sounds like it's not specifically testing UI code.

My concern is that we might be losing coverage of the UI for cloning and that it's not obvious that `"should be cloned without copying the published status"` covers it if I were looking for it.